src/ripple/app/misc/impl/ValidatorSite.cpp

```diff
@@ -231,6 +232,9 @@ ValidatorSite::onSiteFetch(
                 body["signature"].asString(),
                 body["version"].asUInt());
 
+            sites_[siteIdx].lastRefreshStatus.emplace(
+                Site::Status{clock_type::now(), disp});
```

Comment:
I think `lastRefreshStatus` should also be updated with `ListDisposition::invalid` in the `else` below in the case that we fail to parse the JSON.